packages/console/src/components/DragDrop/DraggableItem.tsx

```diff
@@ -11,6 +11,7 @@ type Props = {
   sortIndex: number;
   moveItem: (dragIndex: number, hoverIndex: number) => void;
   children: ReactNode;
+  dragType?: string;
```

Comment:
Would be better to make it a string union?

Reply:
It's used by the user, and it is hard to determine what value to choose.